helpers/html.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require 'sinatra/base'
+
+module Sinatra
+  module HTMLEscapeHelper
+    def h(text)
```

Comment:
Ah, this was copied from [the official documentation](http://www.sinatrarb.com/extensions.html#extending-the-request-context-with-sinatrahelpers), I can certainly rename it